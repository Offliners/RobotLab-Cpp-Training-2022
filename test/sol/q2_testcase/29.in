288
sEeK
prudent
discipline
cluster
ofFEr
thiRSty
EXTRA
INTElLECt
gang
DEPLOY
ConCEIVE
RiCH
enviSagE
A9r
neaRly
patiEnt
s516e452
PriNCE
EXTERIOR
UNGrATEFUl
RiB
hUmOrOusly
RAINCOAT
MYstErY
mAnufaCTURer
CroCodilE
VolUntary
ENTIrE
AGGREGAte
cONVINce
CorpoRatiON
RAnK
coacH
pREVENT
ARIThmETIC
loop
diminish
indignAtIOn
jusT
careFul
break
tReSpass
BeNCH
gaMe
LAST
aCCOuNt
attach
HARNESS
traDe
MidnigHt
PiANiST
faCulty
bond
OVERtAKE
ComfOrtaBLy
MURMuR
cOnvENtioNaL
EAST
ALlIANcE
DiScover
BOOTH
PilgRiM
accompany
coExIsT
FINANCIAL
CoNCEal
ROoSTER
follow
KitcHeN
uNderStanDaBle
advancement
deer
LAdY
SKINNY
cupboard
reprEss
develop
ambush
hOUSEWoRk
integritY
cute
procESS
anNual
preferAbly
imagiNe
MetapHysiCAl
tIE
PRIVATE
tranScend
ForMer
vILLAGE
following
meSsAGE
equal
2R9y
deSPite
coMPEtenCe
AFFAIR
PRECiSe
misery
CaNyON
SPARROW
GIFt
Doubt
mask
eACh
SupERHIghwAy
bet
9572eR
iNtERMEDiaTE
country
uNprediCtable
reUnIoN
CARRIAGE
mOsquito
ACCEPTABLE
StReNgTh
REPutATION
cOmEdY
MIserable
BAlLEt
uSe
CheeR
re84g7104on
huMIlIAte
EyE
influenza
cOunter
EnchAnT
pIETy
tRAcE
CHronIC
BEneAtH
SAtUrDAY
CLOTHE
isOlAtE
MERCY
WHETHEr
DeMand
rETReat
maYor
STUDIO
EValuation
STEWarD
eNRicH
MAYBE
Rice
Confer
bonUs
PoSTAgE
GYPSY
DEPRESSION
inTerruPTion
GOVErNoR
LounGE
DOSE
ENorMOUS
abbrEviatIoN
AgGreSSiOn
lanGUaGe
HARASSMENT
fATHer
EXamiNEE
ISoLatEd
At7r4CT1ve
PRESENCe
Lid
conSEnsus
EXPLORE
ACREAGE
REPLy
integral
spEcTra
SILK
COuNCILmAN
potenT
SOLVE
cAuTIOUS
ReHEarSAl
mare
IMPaTIeNT
BESt
ABRuPT
FIx
alienAtion
ACe
bURGLar
Bug
back
SoCIAL
POND
CouRtESY
Pr5udlY
DUsT
ENJoYaBLE
TRANSLATOR
dIsgUiSE
UNLOAD
vIABLe
eXhibit
chIlDreN
IdeoLoGICaL
monkey
BADMINTOn
Sky
FROST
miSFORTuNe
huNge3
5P77719
conTeNTeD
EASTWarDS
cOnsenT
LEAf
consOlidatE
oBViOUS
ILLuSTrATION
COnTaIneR
Man
ESTEEM
HUmblY
c0ec1-02
EXAg3Era7E
GrAVity
leak
carp
ARROW
ThUNDER
rAlLY
mARket
wEALTHY
eLIMINATe
onE
lateLY
sensiTivE
diScLOSe
basin
LOVING
appLauSE
maRBLe
acUTE
ovErseaS
RaTIoNAl
CarbOn
neSt
BESIDES
5es05ir8NG
LEisure
ARCHiTECT
sIMpLiCity
SwEAR
inspire
EXTENSIVE
ATMOSPHERE
stoP
LOOM
SELL
WET
me434n1e3
efFICiENcY
housewife
bAND
confidently
educated
dinNeR
bleSs
deStinatION
pAUSE
eXaGGerAtion
electRicity
BorRoW
BEGINNER
ADVANCED
DIssIdeNT
suMMit
5APTI6M
doOR
CouRTEOUs
OAK
boX
hundred
gym341889/g5m
BIrD
history
wOlFRAM
VOWEL
erAsE
Pay
LeaD
