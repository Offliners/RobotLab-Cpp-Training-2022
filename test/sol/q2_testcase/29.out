v. 尋覓
adj. 小心謹慎的
n. 紀律
n. 群
n/v. 提議；提供
adj. 口渴的
adj./adv./n. 額外的
n. 理解力
n. 惡少幫；阿飛黨
v./n. 布署
v. 認為；構想
adj. 富有的
v. 預見
Unknown
adv. 幾乎
adj. 有耐心的
Unknown
n. 王子
adj. 外部的
adj. 忘恩負義的
n. 肋骨
adv. 幽默地
n. 雨衣
n. 神祕的事物
n. 製造者
n. 鱷魚
adj. 自願的
adj. 全部的
n. 總數
v. 說服
n. 公司
n./v. 等級；並排
n./v. 教練員；公車
v. 預防
adj./n. 算術的
n. 環形花狀
v. 減少
n. 憤慨
adj. 公平的
adj. 仔細的
n./v. 打破；破曉
v. 擅入
n. 長板凳
n. 遊戲；競賽
adj./adv./n./pron.  最後的；最近的
n. 帳戶；描述
v.(n. ) 裝上；附加
n. 馬具
n./v. 貿易
n. 午夜
n. 鋼琴家
n. 官能；技能
n. 聯結；契約；債券
v. 追上
adv. 舒適地
n./V. 私語
adj. 慣例的；協定上的
adj./adv./n. 東方的
n. 聯盟
v. 發現
n. 售貨棚
n. 朝聖者
v. 陪同
v. 同時共存
adj. 財政的；金融的
v. 隱藏
n. 公雞
v. 跟隨
n. 廚房
adj. 可理解的
n. 前進；擢升
n. 鹿
n. 女士
adj. 消瘦的
n. 碗櫃
v. 抑制
v.(n. )發展
v. 埋伏突擊
n. 家事；家務
n. 廉正
adj. 可愛的
n./v. 過程；加工
adj. 一年一度的
adv. 更好地
v. 想像
adj. 玄學的；形而上學的
n. 領帶
adj. 個人的
v. 超越
adj. 先前的；從前的
n. 村莊
adj./n. 接著的
n. 消息
adj./n./v. 相等的
Unknown
prep. 儘管
n. 能力
n. 事情；事件
adj. 精確的
n. 不幸；悲慘
n. 峽谷
n. 麻雀
n. 禮品；天賦
v./n. 懷疑；疑點
n./v. 面具
adj./adv./pron. 各個
n. 超級高速公路
n./v. 打賭
Unknown
adj. 中問的
n. 國家；鄉下
adj. 不可預測的
n. 重新結合；聚會
n. 四輪馬車；運輸
n. 蚊
adj. 可以接受的
n. 力氣
n. 名聲
n. 喜劇
adj. 不幸的
n. 芭蕾舞
n./v. 益處；使用
n./v. 歡呼；激勵
Unknown
v. 屈辱
n./v. 眼睛
n. 流行性感冒
n. 櫃檯；籌碼
v. 使入迷
n. 虔誠
n./v. 蹤跡；追溯
adj. 慢性的
prep. 在…下面
n. 星期六
v. 為…穿衣
v. 使孤立
n. 慈悲
conj. 是否
n./v. 請求；要求
v. 後退；撒退
n. 市長；鎮長
n. 工作室
n. 估算
n. 服務員
v. 使富裕
adv. 可能
n. 米
v. 商議；授予
n. 紅利
n. 郵資
n. 吉普賽
n. 沮喪
n. 阻礙；干擾
n. 州長；主管
n./v. 休息室；懶洋洋地坐或站
n. (藥物等的)一劑
adj. 巨大的
n. 縮語
n. 侵略
n. 語言
n. 煩惱
n. 父親
n. 應試者
adj. 孤立的
Unknown
n. 出席
n. 蓋子；眼瞼
n. 多數人的意見
v. 探索
n. 英畝數
n./v. 答覆
adj. 構成整體所需要的
n. 光譜
n. 絲
n. 議員
adj. 有效力的
v. 解答
adj. 小心的
n. 排練；複述
n. 母馬；母驢
adj. 不耐煩的
adj./adv. 最好的
adj. 突然的；魯莽的
v. 修理；校準
n. 離間；疏遠
n. 一點(撲克牌)；能手
n. 竊賊
n. 蟲子
adj./adv./n./v.  後面的；背部；贊助
adj. 社會的
n. 池塘
n. 有禮的舉止
Unknown
n./v. 灰塵；打掃
adj. 有樂趣的
n. 翻譯者
n./v. 偽裝
v. 從…卸下(貨物)
adj. 切實可行的
v./n. 展示
n. [pl. ]小孩子
adj. 意識形態的
n. 猴子
n. 羽毛球
n. 天空
n./v. 霜；凍
n. 壞運氣；不幸
Unknown
Unknown
adj. 滿足的
adv. 向束
n./v. 同意
n. 葉子；一張或一頁
v. 鞏固
adj. 明顯的
n. 圖解；說明?
n. 容器
n. 男人
n./v. 尊敬
adv. 謙虛地
Unknown
Unknown
n. 重力；萬有引力
v. 漏
n. 鯉魚
n. 箭
n./v. 雷聲；打雷
v./n. 集合；開玩笑
n./v. 市場
adj. 富裕的
v. 消滅；排除
adj./n. 一
adv. 最近
adj. 敏感的
v. 露出
n. 臉盆；洗滌槽
adj. 愛的
n. 掌聲
n. 大理石
adj. 敏銳的
adj./ad v. 來自海外的
adj. 理性的
n.  碳
n./v. 築巢
prep./adv. 除此之外
Unknown
adj./n. 空閒的
n. 建築師；設計師
n. 樸素
v. 發誓
v. 鼓舞
adj. 廣大的
n. 氣壓；氣氛
n./v. 車站；停止
v. 隱約的或威脅性的出現
v. 賣
adj. 溼的
Unknown
n. 效率
n. 家庭主婦
n. 帶子；樂團
adv. 確信地
adj. 受過教育的
n. 晚餐
v./n. 為…祝福
n. 目的地
n./v. 暫停
n. 誇張
n. 電；電流
v. 借入
n. 初學者
adj. 先進的；高等的
adj./n. 持異議的(人)
n. 山頂
Unknown
n. 門
adj. 有禮貌的
n. 橡樹
n./v. 箱，盒；把…裝箱
adj./n. 一百
Unknown
n. 鳥
n. 歷史
Unknown
n. 母音
v. 擦掉；抹去
n/v. 付錢
v. 領導
